docs/ELRS_TX_MODULES.md

```diff
@@ -0,0 +1,29 @@
+# mLRS Documentation: ELRS Tx Modules #
+
+([back to main page](../README.md))
+
+mLRS supports some few ELRS Tx modules, but only for SiK telemetry use. 
+
+> [!IMPORTANT]
+> Only for SiK telemetry use means that RC control is not supported, neither via the JR bay nor via an IN port. This also means that the mLRS Lua script cannot be used, nor can the Yaapu app.
```

Comment:
Should we specify here that configuration should be done by the OLED?  It seems for the 'supported' devices that is what we recommend.

Reply:
I've added a respective sentence.